terraform/validator-sets/dev/node.config.toml

```diff
@@ -1,81 +1,22 @@
-[base]
-peer_id = "<UNUSED>"
-data_dir_path = "<USE_TEMP_DIR>"
-trusted_peers_file = "/opt/libra/etc/trusted_peers.config.toml"
-peer_keypairs_file = "/opt/libra/etc/peer_keypairs.config.toml"
-node_sync_batch_size = 1000
-node_sync_retries = 3
-node_sync_channel_buffer_size = 10
-node_async_log_chan_size = 256
-
-[metrics]
-dir = ""
-collection_interval_ms = 1000
-push_server_addr = ""
-
-[execution]
-address = "localhost"
-port = 59622
-testnet_genesis = false
-genesis_file_location = "/opt/libra/etc/genesis.blob"
-
-[admission_control]
-address = "0.0.0.0"
-admission_control_service_port = 30307
-need_to_check_mempool_before_validation = false
+[network]
+advertised_address = "/ip4/SELF_IP/tcp/30303"
 
 [debug_interface]
-admission_control_node_debug_port = 30306
-storage_node_debug_port = 49125
-secret_service_node_debug_port = 50316
-metrics_server_port = 14297
 address = "0.0.0.0"
+admission_control_node_debug_port = 30306
```

Comment:
Do you want to change this in defaults as well?
It's something else there now

Reply:
I want to sort out all the ports at some point, I'll leave it for now.